### ISSUE TITLE
memory usage too high 

### ISSUE THREAD

**Issue by Akityo** [neuvector/neuvector#170]
**Environment**
Platform: [e.g. Kubernetes, OpenShift, Rancher, Managed k8s (AKS, EKS, GKE, IKS, ...), ...]
Kubernetes/Platform Version(s): 

1.20.4-aliyun.1

**Describe the bug**
A clear and concise description of what the bug is.  (If you are encountering a deployment error, check out the README or navigate to https://open-docs.neuvector.com for full deployment documentation.  Incorrect runtime is the most common reason for Controller and Enforcer CrashLoopBackOff events.)

![image](https://user-images.githubusercontent.com/16974004/162183426-10673332-3937-4cee-8ea6-6244ebab7a50.png)

memory usage too high 

**To Reproduce**

```2022-04-07T10:36:44.14 |INFO|AGT|main.(*Bench).doDockerContainerBench: Running benchmark checks done
2022-04-07T10:36:44.15 |INFO|AGT|main.(*Bench).doContainerCustomCheck: Running benchmark checks done
2022-04-07T10:37:07|DEBU|dp0|dp_data_thr: epoll error: /host/proc/3631255/ns/net-eth0
2022-04-07T10:37:20.452|ERRO|AGT|osutil.getCGroupSocketTable: open net/tcp,udp - error=open /proc/3632603/root/proc/1/net/tcp: no such file or directory
2022-04-07T10:37:20.452|ERRO|AGT|osutil.getCGroupSocketTable: open net/tcp,udp - error=open /proc/3632603/root/proc/1/net/tcp6: no such file or directory
2022-04-07T10:37:20.452|ERRO|AGT|osutil.getCGroupSocketTable: open net/tcp,udp - error=open /proc/3632603/root/proc/1/net/udp: no such file or directory
2022-04-07T10:37:20.453|ERRO|AGT|osutil.getCGroupSocketTable: open net/tcp,udp - error=open /proc/3632603/root/proc/1/net/udp6: no such file or directory
2022-04-07T10:37:47.541|ERRO|AGT|main.updateContainerNetworks: Error reading container network endpint - container=lb-none endpoint=none-endpoint error=Not found
2022-04-07T10:38:08.115|ERRO|AGT|osutil.getCGroupSocketTable: open net/tcp,udp - error=open /proc/3631255/root/proc/1/net/tcp: no such file or directory
2022-04-07T10:38:08.115|ERRO|AGT|osutil.getCGroupSocketTable: open net/tcp,udp - error=open /proc/3631255/root/proc/1/net/tcp6: no such file or directory
2022-04-07T10:38:08.115|ERRO|AGT|osutil.getCGroupSocketTable: open net/tcp,udp - error=open /proc/3631255/root/proc/1/net/udp: no such file or directory
2022-04-07T10:38:08.115|ERRO|AGT|osutil.getCGroupSocketTable: open net/tcp,udp - error=open /proc/3631255/root/proc/1/net/udp6: no such file or directory
2022-04-07T10:38:08.194|INFO|AGT|main.taskAddContainer: - id=ae98a232623f98f38c5df29b3a9f44a8382b70f3d7a3edda4367ee56046489e0 name=b9555b6dd-zjp78_871538d8-8e9b-449a-8255-6d7e530bcca3_0
2022-04-07T10:38:20.718|INFO|AGT|main.(*Bench).doDockerContainerBench: Running benchmark checks done
2022-04-07T10:38:20.725|INFO|AGT|main.(*Bench).doContainerCustomCheck: Running benchmark checks done
2022-04-07T10:38:22.663|INFO|AGT|main.(*TaskScanner).scanSecretLoop: SCRT: in progress ...
2022-04-07T10:38:35.739|INFO|AGT|system.(*SystemTools).CGroupMemoryStatReset: - threshold=2080374784 usage=4770603008
2022-04-07T10:38:35.739|ERRO|AGT|system.(*SystemTools).CGroupMemoryStatReset.func1: - err=write /sys/fs/cgroup/memory/memory.force_empty: device or resource busy
2022-04-07T10:38:36.663|INFO|AGT|main.(*TaskScanner).scanSecretLoop: SCRT: done - Finished=1 TimeUsed=13.999985626s
2022-04-07T10:39:44.852|INFO|AGT|main.taskStopContainer: - c.pid=3632603 container=3ffa5f5f4db8eb3550cb757b0df154b3c0dda18b2af5d44b0f7e335edd75e401 pid=0
2022-04-07T10:39:44.853|ERRO|AGT|main.taskStopContainer: Failed to read container. Use cached info. - error=No such container: 3ffa5f5f4db8eb3550cb757b0df154b3c0dda18b2af5d44b0f7e335edd75e401
 id=3ffa5f5f4db8eb3550cb757b0df154b3c0dda18b2af5d44b0f7e335edd75e401
2022-04-07T10:39:45.088|ERRO|AGT|osutil.getCGroupSocketTable: open net/tcp,udp - error=open /proc/3632603/root/proc/1/net/tcp: no such file or directory
2022-04-07T10:39:45.088|ERRO|AGT|osutil.getCGroupSocketTable: open net/tcp,udp - error=open /proc/3632603/root/proc/1/net/tcp6: no such file or directory
2022-04-07T10:39:45.088|ERRO|AGT|osutil.getCGroupSocketTable: open net/tcp,udp - error=open /proc/3632603/root/proc/1/net/udp: no such file or directory
2022-04-07T10:39:45.088|ERRO|AGT|osutil.getCGroupSocketTable: open net/tcp,udp - error=open /proc/3632603/root/proc/1/net/udp6: no such file or directory
2022-04-07T10:39:46.071|INFO|AGT|main.taskStopContainer: - c.pid=3631255 container=f59b29c5c55b7d011f102b050867f2a072357495dbadc9cc419855b47d2eed83 pid=0
2022-04-07T10:39:46.071|ERRO|AGT|main.taskStopContainer: Failed to read container. Use cached info. - error=No such container: f59b29c5c55b7d011f102b050867f2a072357495dbadc9cc419855b47d2eed83
 id=f59b29c5c55b7d011f102b050867f2a072357495dbadc9cc419855b47d2eed83
2022-04-07T10:39:56.127|INFO|AGT|main.(*Bench).doDockerContainerBench: Running benchmark checks done
2022-04-07T10:39:56.13 |INFO|AGT|main.(*Bench).doContainerCustomCheck: Running benchmark checks done```

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**UI Issue**
Desktop (please complete the following information):**
 - OS: [e.g. MacOs 12.1, Windows 10]
 - Browser [e.g. chrome, safari]
 - Version [e.g. 22]

**Additional context**
Add any other context about the problem here.

**Comment by garyduan** [comment]
Please provide the following informations,
What NeuVector version do you use? The size of the host where the enforcer is running, CPU/memory, how many pods?

**Comment by smangelkramer** [comment]
I can confirm that all known NV v5.0.x releases eats up a huge amount of memory.

In my lab environments with K3S and RKE2, the memory requirement for NeuVector increases permanently over time. I have noticed this recently.

For this reason, I have integrated the namespaces with the NeuVector deployments in the lab into our monitoring.

I will keep an eye on this and report back.

This is, what my memory consumption in my playground looks like over the time:

<img width="691" alt="Bildschirm­foto 2022-11-20 um 16 22 17" src="https://user-images.githubusercontent.com/18002552/202910656-40c59deb-3a83-4cc0-b0fa-b3fa462bf459.png">

My playground:

- SingleNode K3S Cluster with `v1.24.7+k3s1`
- MultiNode RKe2 Cluster with `v1.23.10+rke2r1`
- NeuVector `5.0.1-4`
